charts/camunda-platform/test/unit/console/golden/deployment.golden.yaml

```diff
@@ -69,6 +69,7 @@ spec:
               value: console
             - name: CAMUNDA_CONSOLE_CONTEXT_PATH
               value: ""
+          envFrom:
```

Comment:
Let's have an if condition for the components that don't have preconfigure `envFrom` so we keep the golden/rendered files clean.

Reply:
Moved the if lines above `envFrom` the golden files are set now.